CHANGELOG.md

```diff
@@ -18,16 +18,23 @@ All user visible changes to this project will be documented in this file. This p
         - Duplicating read users avatars under messages. ([#1243])
         - Invalid message's author being displayed sometimes. ([#1243], [#1050])
         - Gallery image flashing when being opened. ([#1246])
+        - Pop-up window button has been removed. ([#1250])
     - Chats tab:
         - Infinite typing indicator occurring sometimes. ([#1243], [#1244])
     - Chat info page:
         - Infinite loader displayed under members list. ([#1246])
+    - User page:
+      - Duplicate Gapopa id changed. ([#1250])
+      - add chat control buttons. ([#1250])
```

Comment:
Why is the first line here capitalized and the second line is lowercase? Every line in the changelog is capitalized, shouldn't the lowercased one be as well capitalized? Why did you decide it to be lowercase?